File: 0205-isomorphic-strings/0205-isomorphic-strings.java
class Solution {
    public boolean isIsomorphic(String s, String t) {
        int[] count_s = new int[200];
        int[] count_t = new int[200];
        if(s.length() != t.length()){
            return false;
        }
        for(int i=0;i<s.length();i++){
            if(count_s[(int) s.charAt(i)]!=count_t[(int) t.charAt(i)]){
                return false;
            }
            count_s[(int) s.charAt(i)] = i + 1;
            count_t[(int) t.charAt(i)] = i + 1;
        }
        return true;
    }
}

File: 0205-isomorphic-strings/0205-isomorphic-strings.py
class Solution:
    def isIsomorphic(self, s: str, t: str) -> bool:
        count_s = [0] * 200 
        count_t = [0] * 200
        if len(s) != len(t):
            return False
        for i in range(0, len(s)):
            if count_s[ord(s[i])] != count_t[ord(t[i])]:
                return False
            count_s[ord(s[i])] = i + 1
            count_t[ord(t[i])] = i + 1
        return True